AIRobot.md

```diff
@@ -0,0 +1 @@
+You've been thinking of adopting one of those new AI robofriends. Yes, that's right. I know you've been lonely of late. [Here's an early present then.](nagma-venture/AIattacks.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.